assets/i18n/en.toml

```diff
@@ -4,26 +4,26 @@ docs = "Documentation"
 blog = "Blog"
 github = "GitHub"
 login = "Vehicle logins"
-about = "About evcc"
+about = "About"
 
 [header.theme]
 auto = "Design: system"
 light = "Design: light"
 dark = "Design: dark"
 
 [footer.version]
-availableShort = "update"
-availableLong = "update available"
-modalTitle = "Update available"
-modalUpdateStarted = "Evcc will restart after the update.."
-modalInstalledVersion = "Currently installed version"
-modalNoReleaseNotes = "No release notes available. More information about the new version can be found here:"
+availableShort = "install"
+availableLong = "new version available"
+modalTitle = "New version available"
+modalUpdateStarted = "Restarting after installing the new version…"
```

Comment:
```suggestion
modalUpdateStarted = "Starting the new version of evcc…"
```

Reply:
Is {version} not available to make it transparent to users?

"Starting version {version} of evcc…"

Reply:
@comradekingu picked you last suggestion. We could add the version number, but I think its clear enough without it.